db/post_migrate/20210714173022_correctly_move_assignments_from_custom_fields_to_a_table.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class CorrectlyMoveAssignmentsFromCustomFieldsToATable < ActiveRecord::Migration[6.1]
+  def up
+    execute <<~SQL
```

Comment:
Maybe add a comment to explain what this query is doing? It would be a bit confusing without the context. Something like

```suggestion
    # An old version of 20210709101534 incorrectly imported `assignments` with
    # the topic_id and assigned_to_id columns flipped. This query deletes those invalid records.
    execute <<~SQL
```

Reply:
Oh, good one!